modules/ROOT/pages/enterprise/system-requirements.adoc

```diff
@@ -9,7 +9,7 @@ include::partial$misc/browser_compatibility.adoc[]
 
 include::partial$misc/mobile_platform_compatibility.adoc[]
 
-Visit link:{baseurl}/mobile[{productname} mobile documentation] for information on {productname} mobile setup and configuration.
+Visit xref:mobile.adoc[{productname} mobile documentation] for information on {productname} mobile setup and configuration.
 
 == Supported Application Servers: Self-hosted Enterprise
 
```

Comment:
Below there's an anchor that needs properly converting:
```
+++<a class="anchor" id="premiumpluginsupport">++++++</a>+++
```
This could be tricky to add though given the line below is an include. I can't see `premiumpluginsupport` used anywhere either, so it might just be easier to delete it.

Reply:
ok, I removed it